cli/src/main.rs

```diff
@@ -33,6 +33,13 @@ async fn main() {
     let args = Args::parse();
     match args.command {
         Command::Start(start_args) => {
+            if start_args.verbose {
```

Comment:
@h1alexbel I think that `if verbose { //do logging}` is not the best design solution, i.e. fakehub is a testing tool, and each step should be logged with the `verbose` flag, and this means that we will have to put `if verbose {}` in many places in the a code base that will make it pretty messy. I suggest creating some kind of decorator/proxy/trait that will contain some logic, and this logging entity will decide whether to log or not, wdyt?

Reply:
@l3r8yJ actually it's not. We do `if verbose` only once to subscribe on the log tracings with certain log level, i.e. info or debug. Logging with info or any other level should take us one simple line. WDYT?

Reply:
@h1alexbel okay, got it

Reply:
@h1alexbel got it